src/runtime.c

```diff
@@ -54,7 +54,9 @@
 #ifndef ENABLE_DLOPEN
 #define ENABLE_DLOPEN
 #endif
+#ifdef ENABLE_SETPROCTITLE
```

Comment:
We *always* want to `dlopen` libfuse. This is not a linking "fix", it changes the functionality significantly.